### ISSUE TITLE
Couldn't find valid bootstrap(s)

### ISSUE THREAD

**Issue by mrdziuban** [theam/aws-lambda-haskell-runtime#15]
I've created a lambda using layer v7 in us-east-1, and I'm getting this error when running a test event against it:

```json
{
  "errorType": "Runtime.ExitError",
  "errorMessage": "RequestId: 69b1919f-0971-11e9-bfd5-23465fb3fa95 Error: Couldn't find valid bootstrap(s): [/var/task/bootstrap /opt/bootstrap]"
}
```

My `Lib.hs` is pretty simple:

```hs
module Lib where

import Aws.Lambda.Runtime
import Data.Aeson

handler :: Value -> Context -> IO (Either String Value)
handler v _ = return (Right v)
```

Any help with what I might be doing wrong is appreciated. Thanks in advance!

**Comment by NickSeagull** [comment]
Hey @mrdziuban! Thanks for reaching out :)

How did you assign the layer to the lambda? This error generally pops up when it wasn't properly assigned.

Also, could you please post the ARN to make sure that it is the proper one?
Thanks again :)

**Comment by Kalm004** [comment]
Hi @mrdziuban, I was able to reproduce your problem just following the guide. We'll check it and let you know something ASAP.
AFAICS the version 2 is able to run the example without problems, could you try to run your code with it?
Thanks!

**Comment by surt666** [comment]
I get the same experience in eu-west-1 as y-taka-23

**Comment by NickSeagull** [comment]
This issue is fixed in 194bb8a8d94c1b4bb64bb8e45a027a7fc32d17f6 . Also, we have fixed the versioning issue, so now all regions have the same version. Just use now the name `aws-haskell-runtime` as the layer name (full steps are updated in README).

We have tested it on our side, and it works, please reopen if the issue still persists.

Have an awesome day! :grin: :pray:
